featurebyte/models/offline_store_feature_table.py

```diff
@@ -115,11 +117,12 @@ class OfflineStoreFeatureTableUpdate(BaseDocumentServiceUpdateSchema):
     OfflineStoreFeatureTableUpdate class
     """
 
-    feature_ids: List[PydanticObjectId]
-    feature_cluster: FeatureCluster
-    output_column_names: List[str]
-    output_dtypes: List[DBVarType]
-    entity_universe: WindowAggregateEntityUniverse
+    feature_ids: Optional[List[PydanticObjectId]]
+    feature_cluster: Optional[FeatureCluster]
+    output_column_names: Optional[List[str]]
+    output_dtypes: Optional[List[DBVarType]]
+    entity_universe: Optional[WindowAggregateEntityUniverse]
+    last_materialized_at: Optional[datetime]
```

Comment:
🤔 Thinking whether it is better to have something like:

```
class OfflineStoreFeatureTableUpdate(BaseDocumentServiceUpdateSchema):
    """
    OfflineStoreFeatureTableUpdate class
    """

    feature_ids: List[PydanticObjectId]
    feature_cluster: FeatureCluster
    output_column_names: List[str]
    output_dtypes: List[DBVarType]
    entity_universe: WindowAggregateEntityUniverse

class LastMaterializedUpdate(BaseDocumentServiceUpdateSchema):
    last_materialized_at: datetime

XXXUpdate = Union[OfflineStoreFeatureTableUpdate, LastMaterializedUpdate]
```

Reply:
That is definitely better, will fix.